edgeworkers/Acrobat_DC_web_prod/main.js

```diff
@@ -86,32 +88,33 @@ export async function responseProvider(request) {
   };
 
   const fetchFrictionlessPageAndInlineSnippet = async () => {
-    const [responseStream, responseHeaders, version, widgetVersion] = await fetchFrictionlessPage();
+    const [responseStream, responseHeaders, version, widgetVersion, mobileWidget] = await fetchFrictionlessPage();
 
     if (!verb || !locale || !version || !widgetVersion) {
       throw new Error('Missing metadata');
     }
 
-    const snippet =
-      await fetchResource(`/dc/dc-generate-cache/dc-hosted-${version}/${verb}-${locale}.html`);
-    const snippetHead = snippet.substring(snippet.indexOf('<head>')+6, snippet.indexOf('</head>'));
-    const snippetBody = snippet.substring(snippet.indexOf('<body>')+6, snippet.indexOf('</body>'));
-
-    rewriter.onElement('head', el => {
-      el.append(snippetHead);
-    });
-    rewriter.onElement('div.dc-converter-widget', el => {
-      el.append(`<section id="edge-snippet">${snippetBody}</section>`);
-    });
-
+    if (!(mobileWidget && request.device.isMobile)) {
```

Comment:
might be easier to read w/o the extra parens.  Do we mean that both of these conditions must be false or just one?

Reply:
This update branches the condition `mobileWidget && request.device.isMobile` and otherwise. I feel that it's more readable with `mobileWidget && request.device.isMobile` and `!(mobileWidget && request.device.isMobile)`.